overlays/fixed-2clusters/kustomization.yaml

```diff
@@ -0,0 +1,54 @@
+resources:
+  - ../../release/kubernetes-manifests.yaml
+
+patches:
+  - target: # Set label local: true to every pod, then patch to false for offloaded pods.
+      kind: Deployment
+    patch: |-
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: not-used
+        labels: 
+          local: true
```

Comment:
Would do this directly in the nodeAffinityLocal patch if there is no other reason.